packages/opticss/src/util/cssIntrospection.ts

```diff
@@ -16,20 +15,19 @@ export function walkRules(container: postcss.Container, eachRule: RuleIteratorWi
 }
 
 function _walkRulesWithScope(container: postcss.Container, eachRule: RuleIteratorWithScope, scope: RuleScope) {
-  container.each(node => {
+  container.each((node: postcss.ChildNode) => {
     if (isRule(node)) {
       eachRule(node, scope);
-    } else if (isAtRule(node)) {
+    }
+    else if (isAtRule(node)) {
       if (node.name.includes("keyframes") || node.name.includes("font-face")) {
         // skip it, keyframe and font-face at-rules aren't optimizable.
       } else {
         _walkRulesWithScope(node, eachRule, scope.concat(node));
       }
-    } else if (isContainer(node)) {
-      // tslint:disable-next-line:no-console
-      console.log("warning: container that's not an AtRule encountered: " + inspect(node));
```

Comment:
This case what never hit and didn't seem possible with our current code to ever hit. Am I wrong?

Reply:
It was really just future proofing in case postcss's structure ever changes.